CMakePresets.json

```diff
@@ -123,7 +122,8 @@
             "cacheVariables": {
                 "CMAKE_TOOLCHAIN_FILE": "${sourceDir}/vcpkg/scripts/buildsystems/vcpkg.cmake",
                 "CMAKE_POLICY_DEFAULT_CMP0091": "NEW",
-                "FORCE_BUNDLED_MINIZIP": "ON"
+                "FORCE_BUNDLED_MINIZIP": "ON",
+                "ENABLE_CCACHE": "ON"
```

Comment:
not sure that it's a good idea to enforce on users, otherwise you could simply set default to ON in CML